test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -36,7 +36,7 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
-from tests.functional import TestFunctional, SERVER
+from tests.functional import TestFunctional, SERVER, RSC
```

Comment:
(Again same comment 😄) Why can't just do import * ? because today you added one more import 'RSC' in future someone will add more import instead why not just *, like how we do in other testsuites?